src/data/critical-constants-acentric-factor.ts

```diff
@@ -15,4 +15,616 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
+  },
+
+  Benzonitrile: {
+    name: 'Benzonitrile',
+    formula: 'C7H5N',
+    CAS: '100-47-0',
+    molecularWeight: 103.1213,
+    criticalTemperature: 702.3,
+    criticalPressure: 4.215,
+    criticalVolume: 0.3132,
+    criticalCompressibilityFactor: 0.226,
+    acentricFactor: 0.343214,
+  },
+
+  Benzophenone: {
+    name: 'Benzophenone',
+    formula: 'C13H10O',
+    CAS: '119-61-9',
+    molecularWeight: 182.2179,
+    criticalTemperature: 830,
+    criticalPressure: 3.352,
+    criticalVolume: 0.5677,
+    criticalCompressibilityFactor: 0.276,
+    acentricFactor: 0.501941,
+  },
+
+  BenzylAlcohol: {
+    name: 'Benzyl alcohol',
+    formula: 'C7H8O',
+    CAS: '100-51-6',
+    molecularWeight: 108.13782,
+    criticalTemperature: 720.15,
+    criticalPressure: 4.374,
+    criticalVolume: 0.382,
+    criticalCompressibilityFactor: 0.279,
+    acentricFactor: 0.363116,
+  },
+
+  BenzylEthylEther: {
+    name: 'Benzyl ethyl ether',
+    formula: 'C9H12O',
+    CAS: '539-30-0',
+    molecularWeight: 136.19098,
+    criticalTemperature: 662,
+    criticalPressure: 3.11,
+    criticalVolume: 0.442,
+    criticalCompressibilityFactor: 0.25,
+    acentricFactor: 0.433236,
+  },
+
+  BenzylMercaptan: {
+    name: 'Benzyl mercaptan',
+    formula: 'C7H8S',
+    CAS: '100-53-8',
+    molecularWeight: 124.20342,
+    criticalTemperature: 718,
+    criticalPressure: 4.06,
+    criticalVolume: 0.367,
+    criticalCompressibilityFactor: 0.25,
+    acentricFactor: 0.312604,
+  },
+
+  Biphenyl: {
+    name: 'Biphenyl',
+    formula: 'C12H10',
+    CAS: '92-52-4',
+    molecularWeight: 154.2078,
+    criticalTemperature: 773,
+    criticalPressure: 3.38,
+    criticalVolume: 0.497,
+    criticalCompressibilityFactor: 0.261,
+    acentricFactor: 0.402873,
+  },
+
+  Bromine: {
+    name: 'Bromine',
+    formula: 'Br2',
+    CAS: '7726-95-6',
+    molecularWeight: 159.808,
+    criticalTemperature: 584.15,
+    criticalPressure: 10.3,
+    criticalVolume: 0.135,
+    criticalCompressibilityFactor: 0.286,
+    acentricFactor: 0.128997,
+  },
+
+  Bromobenzene: {
+    name: 'Bromobenzene',
+    formula: 'C6H5Br',
+    CAS: '108-86-1',
+    molecularWeight: 157.0079,
+    criticalTemperature: 670.15,
+    criticalPressure: 4.5191,
+    criticalVolume: 0.324,
+    criticalCompressibilityFactor: 0.263,
+    acentricFactor: 0.250575,
+  },
+
+  Bromoethane: {
+    name: 'Bromoethane',
+    formula: 'C2H5Br',
+    CAS: '74-96-4',
+    molecularWeight: 108.965,
+    criticalTemperature: 503.8,
+    criticalPressure: 5.565,
+    criticalVolume: 0.204,
+    criticalCompressibilityFactor: 0.271,
+    acentricFactor: 0.205275,
+  },
+
+  Bromomethane: {
+    name: 'Bromomethane',
+    formula: 'CH3Br',
+    CAS: '74-83-9',
+    molecularWeight: 94.93852,
+    criticalTemperature: 464,
+    criticalPressure: 6.929,
+    criticalVolume: 0.152,
+    criticalCompressibilityFactor: 0.273,
+    acentricFactor: 0.153426,
+  },
+
+  Butadiene: {
```

Comment:
```suggestion
  '1,2Butadiene': {
```

Don't switch the order of the prefixes.